common/src/models/domain/cipher.ts

```diff
@@ -186,12 +185,11 @@ export class Cipher extends Domain {
     return model;
   }
 
-  toCipherData(userId: string): CipherData {
+  toCipherData(): CipherData {
     const c = new CipherData();
     c.id = this.id;
     c.organizationId = this.organizationId;
     c.folderId = this.folderId;
-    c.userId = this.organizationId != null ? userId : null;
```

Comment:
Are we sure this isn't used? Seems that we use `userId` vs. `organizationId` to determine the ownership of the vault item.

Reply:
It's only available in `CipherData`. Which makes it quite difficult for us to actually use it in our applications 😀.